shared/src/components/keyboard/slider/keyboard-slider.component.html

```diff
@@ -3,6 +3,8 @@
     [moduleConfig]="layer.modules"
     [keybindAnimationEnabled]="keybindAnimationEnabled"
     [capturingEnabled]="capturingEnabled"
+    [selectedKey]="selectedKey"
+    [selected]="selectedKey && selectedKey.layerId === index"
```

Comment:
How about merging the selected and selectedKey into this
[selected]="selected?.layerId === index ? selected : undefined" ?
You could use similar logic at the other places.